History/js/new.js

```diff
@@ -249,6 +249,43 @@ var callbacks = {
       // Clear the old iframe and insert the new one
       $(".privly_iframe").empty();
       $(".privly_iframe").append(iFrame);
+    }
+
+    /** 
+    * Get the reference of button previous to the one who's data is displayed and call iframeReturn function. 
+    * 'buttonClicked' variable basically stores the 'Preview ZeroBin/PlainPost' button whose corresponding
+    * data is being displayed in the lightbox.  
+    */ 
+    $('button#prev_preview').on('click',function(){
+      //finds the previous button with preview_link class and passes its reference.
+      //a small example of why this works : http://jsfiddle.net/5QdgB/
+      var index = $('.preview_link').index(buttonClicked);
+      var previous = $('.preview_link').slice(index-1).first();
+      buttonClicked = previous;
+      iframeReturn(previous);
+    });
+
+    /** 
+    * Get the reference of button next to the one who's data is displayed and call iframeReturn function. 
+    * If the user is currently viewing the last message, and clicks on 'Next'
+    * roll back up to the first message (index = 0) and display that.  
+    */ 
+    $('button#next_preview').on('click',function(){
+      //finds the next button with preview_link class and passes its reference.
+      var index = $('.preview_link').index(buttonClicked);
+      if(index == $('.preview_link').length - 1){
+        index = -1;     //rolling back to the first message after the last.
+      }
+      var next = $('.preview_link').slice(index+1).first();
+      buttonClicked=next;
+      iframeReturn(next);
+    });
+    /** 
```

Comment:
Nit: Inconsistent whitespace in the block comment. All non-inline comments should have a blank line preceding them. This is also violated in one case above.